tests/functional/client.js

```diff
@@ -119,4 +119,15 @@ describe('Bucket Client tests', function testClient() {
             return undefined;
         });
     });
+
```

Comment:
Had a doubt, so I double-checked:
This test-suite is actually never launched automatically, and should be (probably?) launched against a live bucketd. That being said, they're not working, and it seems they've long been forgotten. I will probably remove this test-suite altogether, because it gives a false impression of coverage when the tests are not even ran automatically, and can't really be (unless we create a mockup for bucketd which makes no sense imho)

As such, please consider this code (the test's) as dead-code, and the added route untested (unless E2E proves otherwise).

Reply:
I understand. At least running r+ ensures that we will not break current functionality. I will followup with an E2E test for it soon.